_playwright-tests/UI/CustomRepoPagination.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { test, expect } from '@playwright/test';
+import { navigateToRepositories } from './helpers/navHelpers';
+import { deleteAllRepos } from './helpers/deleteRepositories';
+import { bulkCreateRepos } from './helpers/createRepositories';
+import { closePopupsIfExist } from './helpers/helpers';
+
+const repoNamePrefix = 'custom_repo-pagination';
+
+test.describe('Custom repositories pagination', () => {
+  test('Test pagination on custom repo page', async ({ page }) => {
+    await test.step('Delete any pagination test repos that exist', async () => {
+      await deleteAllRepos(page, `&search=${repoNamePrefix}`);
+      await page.waitForTimeout(1500); // Wait for soft delete
+    });
+    await test.step('Populate the custom repo table', async () => {
+      await bulkCreateRepos(page, 12); // 12 counting from 1
```

Comment:
Unnecessary comment :) (I know why it is there, but now it is not needed)

Reply:
At first it was counting from 0, now changed and removed the comment.